wallet/src/account/mod.rs

```diff
@@ -868,5 +939,66 @@ fn group_utxos_for_input<T, Grouped: Clone>(
     Ok(tokens_grouped)
 }
 
+pub fn tx_size_with_outputs(outputs: &[TxOutput]) -> usize {
+    let tx = SignedTransaction::new(
+        Transaction::new(1, vec![], outputs.into()).expect("should not fail"),
+        vec![],
+    )
+    .expect("should not fail");
+    serialization::Encode::encoded_size(&tx)
+}
+
+fn input_signature_size(destination: &Destination) -> usize {
+    let sig_hex = "003c002dd5ea8f05240394eb109b8e9b52716db2720cebbc5dd66394afdf761d2ef76dc8a292d3a44e4d9f1f8fd8e17dc404e317082f5a2ce8adafde01e766aaa6";
```

Comment:
This isn't really what I meant. In the crypto crate, there's the `Signature` type, which can have different signatures. There should be something like a method there that returns the maximum encoded size of the signature.

Reply:
replaced with precalculated sizes